app/controllers/journey_controller.rb

```diff
@@ -47,4 +43,24 @@ def game
     @location_id = params[:location_id]
     render "journey/game#{@location_id}.html.erb"
   end
+
+  def restart
+    @status = status
+    if @status
+      session[:in_game] = nil
+      current_character.user_id = nil
+      render layout: 'wide',  :locals => {:background => "restart-#{session[:alive]}"}
```

Comment:
Line is too long. [86/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.